packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/utils/overrides-children.ts

```diff
@@ -0,0 +1,33 @@
+import { TailwindHelpers } from '../../../../types';
+
+/**
+ * Overrides the `suggestion` styles when it is inside a `suggestion group`.
+ *
+ * @param helpers
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function overridesChildren(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    '> *:first-child:not(.suggestion-group-button)': {
+      marginInlineStart: theme('spacing.8')
+    },
+    '> *:last-child:not(.suggestion-group-button)': {
+      marginInlineEnd: theme('spacing.8')
+    },
+    '.suggestion': {
+      color: 'currentColor',
+      minHeight: 'inherit',
+      fontSize: 'inherit',
+      fontFamily: 'inherit',
+      fontWeight: 'inherit'
+    },
+
+    '&.suggestion-group-ghost:hover, &.suggestion-group-outlined:hover': {
+      '.suggestion': {
+        textDecoration: 'none'
+      }
+    }
+  };
```

Comment:
suggestion: Wouldn't make more sense to have this defined in the `ghost` and `outlined` variant files? As I told you before I believe it will be harder to maintain if we start to place styles for the same variant in different files rather than in the same file. Moreover these styles would also have less specifity.

Reply:
I just got rid of this util when I adjusted the margin by size variant